src/routes/safe/components/Apps/index.tsx

```diff
@@ -419,11 +422,11 @@ function Apps({ closeModal, closeSnackbar, enqueueSnackbar, openModal }) {
           </LCL.Footer> */}
         </LCL.Wrapper>
       ) : (
-        <Card style={{ marginBottom: '24px' }}>
+        <StyledCard>
           <Centered>
```

Comment:
nitpick: is it possible to use styles from centered for Card, so we get rid of one div?

Reply:
mmm I don't see how.. any idea?

Reply:
move styles from centered to styledcard?

Reply:
not possible, centered is being used in other places